src/main/java/no/nav/pam/annonsemottak/receivers/xmlstilling/StillingRepositoryFacade.java

```diff
@@ -75,4 +83,16 @@ private void mergeWithDb(Stilling stilling) {
                 .ifPresent(stilling::merge);
     }
 
+    /**
+     * @deprecated Fjern når alle solr-stillinger er utløpt. Da er denne ikke akutell lenger
+     */
+    @Deprecated
+    private void mergeWithDbArena(Stilling stilling) {
+        stillingRepository.findByKildeAndMediumAndExternalId(Kilde.STILLINGSOLR.value(), "Overført fra arbeidsgiver", stilling.getArenaId())
+                .ifPresent(dbStilling -> {
```

Comment:
Det skal vel kanskje godt la seg gjøre at den ikke finnes, da den allerede er merket som CHANGED_ARENA på bakgrunn av samme spørring. Hvis ikke burde det kanskje vært logget eller laget en metrikk på at noe ikke er helt som det skal. Men jeg tror det er greit sånn som det er.

Reply:
Er vel mest fordi jeg er allergisk mot intellij-warnings, som jeg får hvis jeg kaller en get() direkte på optional'en

Reply:
Tror ikke det er nødvendig med noe metrikk på det, skal ikke kunne være tom som du sier - men er vanskelig å lagre verdien ved første henting uten å sause sammen elementene i koden mer, og fjerne optional blir krever mye endringer mange steder. Falt derfor på at ifPresent var ok for kortlevd kode